src/kitovu/utils.py

```diff
@@ -22,51 +23,49 @@ def get_password(plugin: str, identifier: str) -> str:
     return password
 
 
-class Error(Exception):
+class SchemaValidator:
+    """A validator for creating and merging errors in schema definitions."""
+
+    def __init__(self, abort: bool = True) -> None:
+        self.errors: typing.List[str] = []
+        self.abort: bool = abort
```

Comment:
I don't understand why we need two ways of signalling errors - can't we always do what we do with `abort=True` and let the caller handle the exception? You already get multiple erors in the `InvalidSettingsErrors` exception (and the message and all), so why duplicate all that functionality again with the `valid` property and `abort=False`?